### PR TITLE
Search on n characters

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Changelog
 
-## [10.6.0] - 2020-02-04
+## [10.7.0] - 2020-02-04
```
Comment (atcampbell) [inline CHANGELOG.md]: Should still be 10.6 if it wasn't published to npm?

File: src/components/TypeaheadTable.js [hunk 2]
```diff
@@ -117,7 +128,9 @@ TypeaheadTable.defaultProps = {
     placeholder: '',
     table: { rows: [] },
     label: '',
-    debounce: null
+    debounce: 500,
+    minimumSearchTermLength: 1,
+    queryString: ''
```
Comment (atcampbell) [inline src/components/TypeaheadTable.js]: Should this have null as default?

File: CHANGELOG.md (hunk 1 above)
Comment (lewisrenfrew) [reply CHANGELOG.md]: good call

File: src/components/TypeaheadTable.js (hunk 2 above)
Comment (lewisrenfrew) [reply src/components/TypeaheadTable.js]: useSearch has a default arg of 500 so null works

**Pull request merged**